spec/operators/windowTime-spec.js

```diff
@@ -123,6 +131,37 @@ describe('Observable.prototype.windowTime', function () {
     expectObservable(result, unsub).toBe(expected, values);
   });
 
+  it('should dispose window Subjects if the outer is unsubscribed early', function (done) {
+    var virtualTimeScheduler = new Rx.VirtualTimeScheduler();
```

Comment:
in all synchronous tests, there's a global `rxTestScheduler` you can use to do this work. It's based off of `VirtualTimeScheduler`, and you don't have to call `flush` on it per-say. (but you could).